WcaOnRails/config/locales/en.yml

```diff
@@ -1565,6 +1570,9 @@ en:
       info_html: "This committee has the role to approve and announce WCA Competitions and ensure such announcements adhere to WCA quality standards."
     wmt:
       info_html: "This team is responsible for developing and managing the WCA Brand, seeking sponsorships, supporting the distribution of WCA Competition Gear, and marketing WCA Merchandise."
+    councils_html: "<u>Councils</u>"
```

Comment:
If the `<u></u>` is something that is going to be shared by all languages (and I think it will be), can you extract it out of the translation key?

Reply:
Sure!